roles/patch/tasks/rac-opatch.yml

```diff
@@ -204,7 +204,7 @@
   tags: rac-opatch,stop-home
 
 - name: rac-opatch | Run GI opatch apply
-  command: "{{ grid_home }}/OPatch/{{ patch.method }} {{ '-ocmrf {{ swlib_unzip_path }}/ocm.rsp' if patch.ocm else '' }}"
+  command: "{{ grid_home }}/OPatch/{{ patch.method }} {{ '-ocmrf {{ swlib_unzip_path }}/ocm.rsp' if patch.ocm else '' }} {{non_rolling_clause}}"
```

Comment:
This is the location where an if statement might make sense.  Multiline commands are fine :-)

Reply:
We are using the format `{{ <var> }}` extensively in the toolkit to carry variables that can be either null or something that needs to get populated.

Reply:
After discussion, was toying with similar thought to move it into rac-opatch.yml.
Will test the same and also remove the lines rendered obsolete from patch.yml.

Reply:
Done & tested as follows:
```
Following commands successfuly prove the codepaththat was changed as working correctly:
nohup ./cleanup-oracle.sh --ora-version 19 --inventory-file ./inventory_files/inventory_syd2_at-00010-svr002 --yes-i-am-sure --ora-disk-mgmt udev --ora-swlib-path /u01/oracle_install --ora-asm-disks sydney2_asm.json --ora-data-mounts sydney2_data_mounts_config.json >> ~/050521-mntrl-instlln/bms-toolkit/logs/apply_patch_cleanup.sydney2 2>&1 &

nohup ./install-oracle.sh --ora-swlib-bucket gs://bmaas-testing-oracle-software --instance-ssh-user ansible --instance-ssh-key ~/.ssh/id_rsa --backup-dest /u01/backups --ora-swlib-path /u01/oracle_install --ora-version 19 --ora-swlib-type gcs --ora-asm-disks sydney2_asm.json --ora-data-mounts sydney2_data_mounts_config.json --cluster-type RAC --cluster-config sydney2_cluster_config.json --ora-data-diskgroup DATA --ora-reco-diskgroup RECO --ora-db-name orcl --ora-db-container false  --no-patch >> ~/050521-mntrl-instlln/bms-toolkit/logs/apply_patch_syd2.install_shiiite_ 2>&1  &


./apply-patch.sh --ora-swlib-bucket gs://bmaas-testing-oracle-software --ora-swlib-path /u01/oracle_install --ora-staging /u01/oracle_install --ora-version 19.3.0.0.0  --ora-db-name orcl --inventory-file inventory_files/inventory_syd2_at-00010-svr002  > ~/050521-mntrl-instlln/bms-toolkit/logs/apply_patch_SYD2_till_latest_19.9 2>&1


Results:
========
[grid@at-00010-svr002 ~]$ $ORACLE_HOME/OPatch/opatch lspatches
31780966;TOMCAT RELEASE UPDATE 19.0.0.0.0 (31780966)
31773437;ACFS RELEASE UPDATE 19.9.0.0.0 (31773437)
31772784;OCW RELEASE UPDATE 19.9.0.0.0 (31772784)
31771877;Database Release Update : 19.9.0.0.201020 (31771877)

OPatch succeeded.
[grid@at-00010-svr002 ~]$ $ORACLE_HOME/OPatch/opatch lsinventory |grep "applied on" -A2
Patch  31780966     : applied on Thu Jul 01 11:35:15 PDT 2021
Unique Patch ID:  23770942
Patch description:  "TOMCAT RELEASE UPDATE 19.0.0.0.0 (31780966)"
--
Patch  31773437     : applied on Thu Jul 01 11:34:45 PDT 2021
Unique Patch ID:  23768260
Patch description:  "ACFS RELEASE UPDATE 19.9.0.0.0 (31773437)"
--
Patch  31772784     : applied on Thu Jul 01 11:33:47 PDT 2021
Unique Patch ID:  23854876
Patch description:  "OCW RELEASE UPDATE 19.9.0.0.0 (31772784)"
--
Patch  31771877     : applied on Thu Jul 01 11:30:46 PDT 2021
Unique Patch ID:  23869227
Patch description:  "Database Release Update : 19.9.0.0.201020 (31771877)"

###########
[oracle@at-00010-svr002 ~]$ $ORACLE_HOME/OPatch/opatch lsinventory |grep "applied on" -A2
Patch  31668882     : applied on Thu Jul 01 11:44:33 PDT 2021
Unique Patch ID:  23790068
Patch description:  "OJVM RELEASE UPDATE: 19.9.0.0.201020 (31668882)"
--
Patch  31772784     : applied on Thu Jul 01 11:29:32 PDT 2021
Unique Patch ID:  23854876
Patch description:  "OCW RELEASE UPDATE 19.9.0.0.0 (31772784)"
--
Patch  31771877     : applied on Thu Jul 01 11:27:45 PDT 2021
Unique Patch ID:  23869227
Patch description:  "Database Release Update : 19.9.0.0.201020 (31771877)"
[oracle@at-00010-svr002 ~]$ 
[oracle@at-00010-svr002 ~]$ 
[oracle@at-00010-svr002 ~]$ $ORACLE_HOME/OPatch/opatch lspatches
31668882;OJVM RELEASE UPDATE: 19.9.0.0.201020 (31668882)
31772784;OCW RELEASE UPDATE 19.9.0.0.0 (31772784)
31771877;Database Release Update : 19.9.0.0.201020 (31771877)

OPatch succeeded.
```